hoomd/md/methods/thermostats.py

```diff
@@ -62,11 +62,26 @@ class MTTK(Thermostat):
     these quantities using `translational_dof` and
     `rotational_dof`.
 
+    Note:
+        The coupling constant `tau` should be set within a
+        reasonable range to avoid abrupt fluctuations in the kinetic temperature
+        and to avoid long time to equilibration. The recommended value for most
+        systems is :math:`\tau = 100 \delta t`.
+
     See Also:
         `G. J. Martyna, D. J. Tobias, M. L. Klein 1994
         <http://dx.doi.org/10.1063/1.467468>`_ and `J. Cao, G. J. Martyna 1996
         <http://dx.doi.org/10.1063/1.470959>`_.
 
+    Example::
+
+        # NVT integration using MTTK Nose-Hoover thermostat
+        dt = 0.005
+        MTTK = hoomd.md.methods.thermostats.Berendsen(kT=1.0, tau=dt*100)
+        nvt = hoomd.md.methods.ConstantVolume(filter=hoomd.filter.All(),
+                                            thermostat=MTTK)
+        integrator = hoomd.md.Integrator(dt=dt, methods=[nvt], forces=[lj])
```

Comment:
There are no forces defined. Perhaps we should remove that part of the example.
```suggestion
        integrator = hoomd.md.Integrator(dt=dt, methods=[nvt])
```